### PR TITLE
Add IMDB reviews example

### PR DESCRIPTION
**Pull request by seanmor5**
There are some stability issues during training that I am investigating. Hopefully not due to grad :D 

Performance wise this isn't bad!

### CODE REVIEW

File: notebooks/imdb_sentiment_analysis.livemd [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## Data
+
+```elixir
+%{review: reviews, sentiment: sentiment} = Scidata.IMDBReviews.download()
+
+max_seq_len = 64
+
+pad_or_slice_to_length = fn x ->
+  # This will be slow in a JIT :(
```
Comment (josevalim) [inline notebooks/imdb_sentiment_analysis.livemd:22]: Why this will be slow in a JIT? It seems this could be writable as defn?

Comment (seanmor5) [reply notebooks/imdb_sentiment_analysis.livemd:22]: We are trying to normalize the data to a single shape, so its shape will change every time

File: notebooks/imdb_sentiment_analysis.livemd [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+```
+
+## Model
+
+```elixir
+model =
+  Axon.input({nil, max_seq_len})
+  |> Axon.embedding(256, 256)
+  |> Axon.gru(64)
+  |> then(fn {_state, seq} -> seq end)
```
[the remark this answers is not in the document]
Comment (josevalim) [reply notebooks/imdb_sentiment_analysis.livemd:69]: I was going to say correct but then I realized I didn't write the code. :P

[the remark this answers is not in the document]
Comment (seanmor5) [reply notebooks/imdb_sentiment_analysis.livemd:69]: Lol, I've gotten so addicted to `then` I completely forgot

**Comment by seanmor5** [comment]
Closing as stale, I have a better example to push up soon :)

**Pull request closed without merging**